package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
-    "lint": "xo",
+    "lint": "eslint --ext .js,.json . --fix",
```

Comment:
typically autofix is applied explicitly, not implicitly; we wouldn’t want CI to pass when there’s fixable warnings in the code.